packages/dashboard/src/Dashboard.tsx

```diff
@@ -1,10 +1,6 @@
-import { lazy, Suspense } from 'react'
+import { lazy } from 'react'
 
 const Dashboard = lazy(() => import(/* webpackPreload: true */ './initialization/Dashboard.js'))
 export function IntegratedDashboard() {
-    return (
```

Comment:
Why remove suspense

Reply:
The `Maskbook/packages/dashboard/src/pages/routes.tsx` has suspense. 
And the root suspense lead to the issue: #mf-3185